### PR TITLE
[Codebase] Automatically find or add world

### PR DESCRIPTION
**Pull request by Lilaa3**
Possible solution to #330 
Never changes the actual scene.world, as that should be left to the user, the important logic is in create_or_get_world.
This will almost never create a world because of the f3d nodes library having its own world

### CODE REVIEW

**Comment by Lilaa3** [comment]
@Dragorn421 could you look at this at some point?

**Comment by Lilaa3** [comment]
Would an exception or warning popup work better?

File: fast64_internal/utility.py [hunk 1]
```diff
@@ -1868,3 +1868,22 @@ def upgrade_old_prop(
...
+    - The first world in bpy.data.worlds if the current file has a world. (Which it almost always does because of the f3d nodes library)
+    - Create a world named "Fast64" and return it if no world exits.
+    This function does not assign any world to the scene.
+    """
+    if scene.world:
+        return scene.world
+    if bpy.data.worlds:
+        world: World = bpy.data.worlds.values()[0]
+        print(f'No world selected in scene, selected the first one found in this file "{world.name}".')
+        return bpy.data.worlds.values()[0]
```
Comment (Dragorn421) [inline fast64_internal/utility.py:1886]:
```suggestion
        return world
```

Comment (Lilaa3) [reply fast64_internal/utility.py]: True

**Review by Dragorn421 — APPROVED** [review]
great!

**Comment by Dragorn421** [comment]
Will fix #330

File: fast64_internal/utility.py [hunk 2]
```diff
@@ -1868,3 +1868,22 @@ def upgrade_old_prop(
...
+    - The world selected in the scene if the scene has a selected world.
+    - The first world in bpy.data.worlds if the current file has a world. (Which it almost always does because of the f3d nodes library)
+    - Create a world named "Fast64" and return it if no world exits.
+    This function does not assign any world to the scene.
+    """
+    if scene.world:
+        return scene.world
+    if bpy.data.worlds:
+        world: World = bpy.data.worlds.values()[0]
+        print(f'No world selected in scene, selected the first one found in this file "{world.name}".')
```
Comment (Dragorn421) [inline fast64_internal/utility.py:1885]: actually I just realized this severely spams the console. Could you add a counter for this print and only print it the first 10 times or something
use a dirty global variable :)

**Review by Dragorn421 — APPROVED** [review]
thanks

**Pull request merged**